public/docs/_examples/gettingstarted/ts/src/tsconfig.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
Right, for the reasons you point out I think its worth in getting started, just to give it to them. GS is about getting moving fat and getting early positive reinforcement. I think its better for now to give it to them.

Otherwise its as you described:
- go to the folder
- init
- change settings
- go back a folder

that is friction i would rather avoid in GS.